### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by sobrase**

### CODE REVIEW

File: roles/core/nfs_server/tasks/main.yml [hunk 1]
```diff
@@ -20,6 +20,12 @@
 
 - meta: flush_handlers
 
+- name: Create directory                                                                                                                                                                                             file:
+    path: "{{ nfs[item]['export'] }}"
```
Comment (actatux) [inline roles/core/nfs_server/tasks/main.yml]: I guess it should be nfs[item]['mount'] which is the path the NFS server should export?

**Comment by actatux** [comment]
Also, could you describe the change in your commit title and message? There is at least one main.yml per role, so we can't know which role is modified or why it is modified with the current commit message. It would be very helpful when looking at the git log.

eg.: **Feat: Create the NFS directories to export**

Thanks :)

**Comment by oxedions** [comment]
Dear @sobrase ,

I honestly would prefer to skip directory creation on the nfs_server part. While I find it relevant in nfs_client part, I think it can be difficult to handle, or dangerous on the nfs_server part. At least, there must be a variable, false by default, to activate/deactivate this feature :-)

Ben

**Comment by oxedions** [comment]
No activity, and still don't think creating directories automatically is a good idea on server side.

Closing.

**Pull request closed without merging**